meta-lmp-support/recipes-core/initrdscripts/initramfs-framework/ostreecheck

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+# Initial concept of how to do OSTree fsck+diff to check
+# rootfs integrity.
+#
+# TODO:
+# - uncomment "fatal"s
+# - change various "msg" to "info"/"debug"
+# - verify validity of deployment hash via a signature
+# - verify version number to prevent rollback
+# - handling of /etc ?
+
+ostreecheck_enabled() {
+	return 0
+}
+
+ostreecheck_run() {
```

Comment:
@kjbracey-arm , is this script is signed as apart of boot partition?

Reply:
@kjbracey-arm ?

Reply:
It's in initramfs, so should be in the same partition as the kernel.

Reply:
Thanks @kjbracey-arm.
@JanneKiiskila it means that at least this specific script must be verified by uboot before run( of course it is better to verify the entire initramfs )

Reply:
You will definitely want to have a signature for the whole initramfs .img, along with the kernel. This is reducing the problem scope to basically just handling those 2 files, rather than the whole filing system...